src/renderer.zig

```diff
@@ -182,6 +184,8 @@ pub fn renderWorld(world: *World, ambientLight: Vec3f, skyColor: Vec3f, playerPo
 
 	const time: u32 = @intCast(std.time.milliTimestamp() & std.math.maxInt(u32));
 
+	Skybox.render();
```

Comment:
Please add a `gpu_performance_measuring` query for that, so we can see its performance impact in the F5 debug menu.